_layouts/contribute.html

```diff
@@ -22,21 +22,27 @@
               <li><a href="https://github.com/bazelbuild/bazel/wiki/Bazel-Users">Who's Using Bazel</a></li>
               <li><a href="/governance.html">Governance</a></li>
               <li><a href="/roadmap.html">Roadmap</a></li>
-              <li><a href="/naming.html">Naming Bazel projects</a></li>
+              <li><a href="/recommended-rules.html">Recommended rules</a></li>
               <li><a href="/support.html">Support</a></li>
             </ul>
-            <h3>Technical Docs</h3>
+            <h3>Contributors guide</h3>
+            <ul class="sidebar-nav">
+              <li><a href="/basics/getting_started.html">Getting started</a></li>
+              <li><a href="/basics/patching.html">Patch acceptance process</a></li>
+              <li><a href="/designs/index.html">Design documents</a></li>
+              <li><a href="/basics/release-notes.html">Release notes</a></li>
+              <li><a href="/naming.html">Naming Bazel projects</a></li>
+            </ul>
+            <h3>Maintainers guide</h3>
             <ul class="sidebar-nav">
+              <li><a href="/maintaining/maintainers-guide.html">Guide for Bazel Maintainers</a></li>
+              <li><a href="/maintaining/breaking-changes-guide.html">Rolling out breaking changes</a></li>
```

Comment:
Contributors also need to know about it.

Reply:
I moved it up.  OTOH, the only people who can successfully add an incompatible change so far have been Bazel team members.